autogpt_platform/backend/backend/blocks/llm.py

```diff
@@ -909,24 +910,24 @@ def trim_prompt(s: str) -> str:
 
         if input_data.expected_format:
             expected_format = [
-                f'"{k}": "{v}"' for k, v in input_data.expected_format.items()
+                f"{repr(k)}: {repr(v)}" for k, v in input_data.expected_format.items()
             ]
             if input_data.list_result:
                 format_prompt = (
                     f'"results": [\n  {{\n  {", ".join(expected_format)}\n  }}\n]'
                 )
             else:
-                format_prompt = "\n  ".join(expected_format)
+                format_prompt = ",\n|  ".join(expected_format)
 
             sys_prompt = trim_prompt(
                 f"""
-                  |Reply strictly only in the following JSON format:
-                  |{{
-                  |  {format_prompt}
-                  |}}
-                  |
-                  |Ensure the response is valid JSON. Do not include any additional text outside of the JSON.
-                  |If you cannot provide all the keys, provide an empty string for the values you cannot answer.
+                |Reply with pure JSON strictly following this JSON format:
+                |{{
+                |  {format_prompt}
+                |}}
+                |
+                |Ensure the response is valid JSON. DO NOT include any additional text (e.g. markdown code block fences) outside of the JSON.
+                |If you cannot provide all the keys, provide an empty string for the values you cannot answer.
```

Comment:
This gets me from ~40% to ~76% one-shot success rate. Improvement is possible but not currently worth my time. The retry mechanism works well enough that those remaining 24% one-shot failures are not a significant issue.